.travis.yml

```diff
@@ -19,7 +19,7 @@ env:
   matrix:
     - TEST_SUITE=test
     - TEST_SUITE=cucumber
-
+    - TEST_SUITE=default-site
```

Comment:
Your call. I'm fine with it running once? If so, what's the best way to do that?